src/App.js

```diff
@@ -1,18 +1,39 @@
 import React from 'react';
+const programTitle = "Todo List";
+const todoList = [
+  {
+    id: 1,
+    title: "Begin To-Do app",
+  },
+  {
+    id: 2,
+    title: "Checkout new github branch"
+  },
+  {
+    id: 3,
+    title: "Create list"
+  },
+  {
+    id: 4,
+    title: "Display list on page"
+  },
+  {
+    id: 5,
+    title: "Push changes to github"
+  }
+];
 
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div>
+        <h1>{programTitle}</h1>
+        <ul>
+          {todoList.map(function(item) {
+            return <li key={item.id}>{item.title}</li>
+          })}
```

Comment:
Small nitpick for these lines (**doesn't require any changes**):

Typically, you'll see function expressions (fat arrow functions `() => {...}`) used _within_ React components. While they function the same as a traditional function within this context, they have some advantages in that they're **subjectively** a bit easier to write and read:
```jsx
<ul>
  {todoList.map(item => {
    return <li key={item.id}>{item.title}</li>
  })}
</ul>
```
Function expressions also have an advantage of [implicit returns](https://dev.to/samanthaming/es6-arrow-functions-cheatsheet-1cn). Therefore, the above could be simplified down to:
```jsx
<ul>
  {todoList.map(item => <li key={item.id}>{item.title}</li>)}
</ul>
```

Once again, this **doesn't require any changes** , so you're welcomed to keep your code as is.